addons/repair/models/repair.py

```diff
@@ -172,11 +173,32 @@ def _default_picking_type_id(self):
         'Allowed to Reserve Production', compute='_compute_unreserve_visible',
         help='Technical field to check when we can reserve quantities')
 
+    @api.depends('picking_id')
+    def _compute_partner_id(self):
```

Comment:
I think `_compute_partner_id`, `compute_picking_lot_ids` and `compute_picking_product_ids` could be merge into a single compute method (`_compute_fields_from_picking` ?) because they all depend only of `picking_id` and do nothing more than getting a value from the picking or its moves

Reply:
Indeed, I didn't pay attention enough, sorry !